esmvaltool/cmorizers/obs/cmorize_obs_era_interim.ncl

```diff
@@ -157,7 +157,7 @@ begin
 
       ; fx variables are time invariant
       if (l_fx) then
-        fname = input_dir_path + "ERA-Interim_" + NAME(0) + ".nc"
+        fname = input_dir_path + "ERA-Interim_" + NAME(0) + "_.nc"
```

Comment:
No longer needed once [#210](https://github.com/ESMValGroup/ESMValCore/pull/210) is merged.

Reply:
```suggestion
        fname = input_dir_path + "ERA-Interim_" + NAME(0) + ".nc"
```